server/src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -97,7 +100,10 @@
                     });
 
     private static final ObjectParser.NamedObjectParser<ClusterIndexHealth, Void> INDEX_PARSER =
-            (XContentParser parser, Void context, String index) -> ClusterIndexHealth.innerFromXContent(parser, index);
+        (XContentParser parser, Void context, String index) -> ClusterIndexHealth.innerFromXContent(parser, index);
+    static final String CLUSTER_HEALTH_REQUEST_TIMEOUT_DEPRECATION_MSG = "HTTP status code for an internal cluster health timeout " +
+        "will be changed from 408 to 200 in 8.0.0. Set the `es.cluster_health.request_timeout_200` property to `true` to opt in for it.";
```

Comment:
Oh also it's better to extract the property name to a field somewhere and then use the field in messages instead of using the literal string in multiple places, see e.g. `PERMIT_HANDSHAKES_FROM_INCOMPATIBLE_BUILDS_KEY`. Saves worrying about typos and helps make sure we don't accidentally have any messages that refer to a nonexistent setting.

And one more thing: we prefer wrapping literals in `[]` rather than backticks in messages like this, backticks tend to result in copy/paste errors whereas brackets are harder to miss:

```suggestion
        "will be changed from 408 to 200 in 8.0.0. Set the [" + PROPERTY + "] property to [true] to opt in for it.";
```